modules/ocf_ldap/manifests/init.pp

```diff
@@ -120,4 +120,17 @@
         action => 'accept',
       };
   }
+
+  package { ['sasl2-bin']:; }
+  augeas { '/etc/default/saslauthd':
+    context => '/files/etc/default/saslauthd',
+    changes => [
+      'set MECHANISMS \'"kerberos5"\'',
+      'set START \'"yes"\'',
+    ],
+    require => Package['sasl2-bin'],
+  }
+  service { 'saslauthd':
+    ensure => running,
```

Comment:
Generally this is an antipattern with puppet, instead you'd want to just make sure the service is enabled to start on boot and leave it at that. Otherwise you can get pretty long puppet runs or the system won't work on boot unless puppet has run, stuff like that

Reply:
I think it's safe to delete this line, since the `START="yes"` line in defaults should make the service start on boot.